docs/asyncapi.yaml

```diff
@@ -127,42 +138,57 @@ components:
         expectedPaymentDate: "21.11.2025"
         adminName: "Jessica Lrrr"
       default: {}
+      minleght: 0
+      maxlenght: 100
 
     reference:
       type: string
-      description: Unique identifier (UUID) for further operations like status query
-      example: "abc123-uuid-123456"
+      description: Unique identifier for further operations like status query
+      example: "can-2a88-be-b855-a55026844MIX"
+      minleght: 2
+      maxlenght: 100
 
     oneclickunsubsurl:
       type: string
       description: URL for one-click unsubscribe option
       example: "https://unsubscribe.example.com"
+      minleght: 4
+      maxlenght: 250
 
     emailreplytoid:
       type: string
+      format: uuid
       description: Optional ID for the reply-to email address
       example: "8e222534-7f05-4972-86e3-17c5d9f894e2"
 
     specversion:
       type: string
       description: The version of the CloudEvents specification that the event uses
       example: '1.0'
+      minleght: 3
+      maxlenght: 10
 
     type:
       type: string
       description: The type of event in reverse DNS notation
       example: uk.gov.fcp.sfd.notification.request
+      minleght: 3
+      maxlenght: 250
 
     source:
       type: string
       description: The service publishing the event
-      example: ffc-ahwr-claim
+      example: github.com/DEFRA/ffc-ahwr-claim
```

Comment:
We just need the service name, not the url to it.

Reply:
I tried to follow suggestions form cloudEvents, specifically : 

![image](https://github.com/user-attachments/assets/9d3a3c8f-2e30-430c-83f9-4ad87773246f)

to use absolute URI.

Reverting back

Reply:
I can see your point, but I'd say the GitHub source code isn't necessarily in line with it.  Also as messages for Grants etc may not come from GitHub repos we build, could come from CoTS products or plugins too.  So I'd suggest we keep it simple.